cinema/serializer.py

```diff
@@ -0,0 +1,26 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.ModelSerializer):
```

Comment:
If you inherit from ModelSerializer, it will automatically generate a set of fields for you, based on the model. And by default, all the model fields on the class will be mapped to a corresponding serializer fields.